Jenkinsfile

```diff
@@ -221,6 +266,104 @@ pipeline {
                 '''
             }
         }
+
+        stage('dist') {
+            steps {
+                sh '''
+                    set -e +x
+                    source ${PWD}/.envrc
+                    set -x
+                    unset HCF_PACKAGE_COMPILATION_CACHE
+                    rm -f scf-*amd64*.zip
+                    make helm bundle-dist
+                '''
+            }
+        }
+
+        stage('deploy') {
+			when {
+				expression { return params.TEST_SMOKE || params.TEST_BRAIN || params.TEST_CATS }
+			}
+            steps {
+                timeout(time: 2, unit: 'HOURS') {
+                    sh """
+                        kubectl create -f - <<< '{"kind":"StorageClass","apiVersion":"storage.k8s.io/v1","metadata":{"name":"'"${JOB_BASE_NAME}-${BUILD_NUMBER}-"'hostpath"},"provisioner":"kubernetes.io/host-path"}'
+
+                        # Unzip the bundle
+                        rm -rf unzipped
+                        mkdir unzipped
+                        unzip -e \$(ls | grep -e 'scf-*linux-amd64*.zip') -d unzipped
```

Comment:
Do we need `ls` + `grep`? Isn't this just `unzip -e scf-*linux-amd64*.zip -d unzipped`?

Reply:
Fixed.